lib/skate_web/controllers/location_search_controller.ex

```diff
@@ -3,6 +3,15 @@ defmodule SkateWeb.LocationSearchController do
 
   alias Skate.LocationSearch.AwsLocationRequest
 
+  @spec get(Plug.Conn.t(), map()) :: Plug.Conn.t()
+  def get(conn, %{"id" => id}) do
+    seach_fn = Application.get_env(:skate, :location_get_fn, &AwsLocationRequest.get/1)
```

Comment:
typo: would you update this and the other instances in this file that aren't in the changes?

Reply:
ec888e70

I also renamed it to `get_fn` in the `get/2` function to better reflect its meaning.